templates/bundles/DemosPlanCoreBundle/DemosPlanNews/news_admin_edit.html.twig

```diff
@@ -150,27 +150,23 @@
                 }) }}
 
             {% else %}
-                {{ uiComponent('form.row', {
-                    elements: [
-                        {
-                            label: { text: 'citizen.for'|trans },
-                            control: { name: 'r_group_code[]', value: 'GGUEST', checked: guestChecked|default(false) },
-                            id: 'citizenVisibility',
-                            type: 'checkbox',
-                            elementSize: 'medium'
-                        },
-                        {
-                            label: { text: 'invitable_institution.for'|trans },
-                            control: { name: 'r_group_code[]', value: 'GPSORG', checked: publicAgencyChecked|default(false) },
-                            id: 'publicAgencyVisibility',
-                            type: 'checkbox',
-                            elementSize: 'medium'
-                        }
-                    ],
-                    heading: 'visible',
-                    required: true,
-                    isFieldset: true
-                }) }}
+                <fieldset>
+                    <legend class="font-semibold mb-1">{{ 'visible'|trans }}&#42</legend>
+                    <dp-checkbox
```

Comment:
This is my bad, because I suggested to use DpCheckbox, but now I realize that we should use DpCheckboxGroup, which contains exactly the markup you used here (fieldset + legend) :see_no_evil: